packages/components/src/dropdown-menu/stories/index.js

```diff
@@ -1,52 +1,92 @@
-/**
- * External dependencies
- */
-import { text, boolean } from '@storybook/addon-knobs';
-
 /**
  * Internal dependencies
  */
 import DropdownMenu from '../';
+import { MenuGroup, MenuItem } from '../../';
 
 /**
  * WordPress dependencies
  */
-import { menu, arrowUp, arrowDown } from '@wordpress/icons';
+import {
+	menu,
+	arrowUp,
+	arrowDown,
+	chevronDown,
+	more,
+	trash,
+} from '@wordpress/icons';
 
 export default {
 	title: 'Components/DropdownMenu',
 	component: DropdownMenu,
+	argTypes: {
+		className: { control: { type: 'text' } },
+		children: { control: { type: null } },
+		disableOpenOnArrowDown: { control: { type: 'boolean' } },
+		icon: {
+			options: [ 'menu', 'chevronDown', 'more' ],
+			mapping: { menu, chevronDown, more },
+			control: { type: 'select' },
+		},
+		menuProps: {
+			control: { type: 'object' },
+		},
+		noIcons: { control: { type: 'boolean' } },
+		popoverProps: {
+			control: { type: 'object' },
+		},
+		text: { control: { type: 'text' } },
+	},
 	parameters: {
-		knobs: { disable: false },
+		controls: { expanded: true },
+		docs: { source: { state: 'open' } },
 	},
 };
 
-export const _default = () => {
-	const label = text( 'Label', 'Select a direction.' );
-	const firstMenuItemLabel = text( 'First Menu Item Label', 'Up' );
-	const secondMenuItemLabel = text( 'First Menu Item Label', 'Down' );
-	const toggleButtonTootip = boolean(
-		'Show tooltip on a toggle button',
-		true
-	);
+const Template = ( props ) => <DropdownMenu { ...props } />;
 
-	const controls = [
+export const Default = Template.bind( {} );
+Default.args = {
+	label: 'Select a direction.',
+	icon: menu,
+	controls: [
 		{
-			title: firstMenuItemLabel,
+			title: 'First Menu Item Label',
 			icon: arrowUp,
+			// eslint-disable-next-line no-console
+			onClick: () => console.log( 'up!' ),
 		},
 		{
-			title: secondMenuItemLabel,
+			title: 'Second Menu Item Label',
 			icon: arrowDown,
+			// eslint-disable-next-line no-console
+			onClick: () => console.log( 'down!' ),
 		},
-	];
+	],
+	toggleProps: {
+		showTooltip: true,
+	},
```

Comment:
Noting that `showTooltip = true` seems to be the default behavior:

https://github.com/WordPress/gutenberg/blob/aab71ed16657eebb1a02b7aa13f98ea88d8566b9/packages/components/src/dropdown-menu/index.js#L126

Reply:
Yup, done in https://github.com/WordPress/gutenberg/pull/47149/commits/f8747ecf1ac56bb22a5d7a368c76ad884d8edf2f